packages/tsconfig-reference/copy/en/options/rootDir.md

```diff
@@ -39,7 +39,7 @@ By setting `rootDir: "."` in `tsconfig.json`, TypeScript would write this tree:
 ```
 MyProj
 ├── dist
-|   ├── core
```

Comment:
@orta 
I found nits typo in original doc and fix it.
(this character should be not `|` but `│` )

Reply:
Thanks!